libraries/AP_Logger/AP_Logger_MAVLink.cpp

```diff
@@ -405,7 +422,8 @@ void AP_Logger_MAVLink::stats_collect()
         return;
     }
     uint8_t pending = queue_size(_blocks_pending);
-    uint8_t sent = queue_size(_blocks_sent);
```

Comment:
I'm *very* uneasy with changing the meaning of this statistic.  This isn't the number of blocks sent, nothing remotely close to it.  It's the number of blocks waiting acknowledgement from the other end.

You're moving it away from being a direct measurement to some other measure, which I don't like the look of!